src/store/Selectors/GraphData.ts

```diff
@@ -71,24 +80,26 @@ const decorateGraphData = (graphData: any) => {
           });
         }
         decoratedNode.data = { ...elementsDefaults.nodes, ...decoratedNode.data };
-        return decoratedNode;
+        // prettier-ignore
+        return <DecoratedGraphNodeWrapper> decoratedNode;
       });
     }
     if (graphData.edges) {
-      graphData.edges = graphData.edges.map(edge => {
+      decoratedGraph.edges = graphData.edges.map((edge: GraphEdgeWrapper) => {
         const decoratedEdge = { ...edge };
+        const { traffic, ...edgeData } = edge.data;
         // see comment above about the 'traffic' data handling
-        if (edge.data.traffic) {
-          const traffic = edge.data.traffic;
+        if (traffic) {
           edge.data.traffic = undefined;
-          decoratedEdge.data = { protocol: traffic.protocol, ...traffic.rates, ...decoratedEdge.data };
+          decoratedEdge.data = { ...traffic.rates, ...edgeData };
```

Comment:
Am I reading this wrong or did you stop setting the protocol field?

Reply:
Ups... my mistake! I'll fix it.

Reply:
Fixed!